model/transformers/tests/test_classification_integration.py

```diff
@@ -0,0 +1,180 @@
+"""
+This file contains integration tests. We use the CLI to exercise functionality of
+various DFFML classes and constructs.
+"""
+import csv
+import json
+import random
+import pathlib
+import contextlib
+
+from dffml.cli.cli import CLI
+from dffml.service.dev import Develop
+from dffml.util.asynctestcase import IntegrationCLITestCase
+
+
+class TestHFClassifier(IntegrationCLITestCase):
+    async def test_run(self):
+        self.required_plugins("dffml-model-transformers")
+        # Randomly generate sample data
+        POSITIVE_WORDS = ["fun", "great", "cool", "awesome", "rad"]
+        NEGATIVE_WORDS = ["lame", "dumb", "silly", "stupid", "boring"]
+        WORDS = [NEGATIVE_WORDS, POSITIVE_WORDS]
+
+        SENTENCES = [
+            "I think my dog is {}",
+            "That cat is {}",
+            "Potatoes are {}",
+            "When I lived in Wisconsin I felt that it was {}",
+            "I think differential equations are {}",
+        ]
+
+        DATA = []
+
+        for example in SENTENCES:
+            sentiment_words = random.choice(WORDS)
+            sentiment_classification = WORDS.index(sentiment_words)
+            DATA.append(
+                [
+                    example.format(
+                        *random.sample(sentiment_words, example.count("{}"))
+                    ),
+                    str(sentiment_classification),
+                ]
+            )
+        data_filename = self.mktempfile() + ".csv"
+        with open(pathlib.Path(data_filename), "w+") as data_file:
+            writer = csv.writer(data_file, delimiter=",")
+            writer.writerow(["text", "sentiment"])
+            writer.writerows(DATA)
+
+        # Features
+        features = "-model-features text:str:1".split()
+
+        # Train the model
+        await CLI.cli(
```

Comment:
It looks like the `.sh` files are never tested. Instead of creating another test file for them. Maybe this is a good place to read them in an use them as the arguments to `CLI.cli`